.github/workflows/lint_python.yml

```diff
@@ -7,20 +7,15 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - run: pip install --upgrade pip wheel
-      - run: pip install bandit black codespell darker flake8 flake8-2020 flake8-bugbear
-                         flake8-comprehensions isort mypy pytest pyupgrade safety
-      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
-      - run: black --check .
-      - run: darker --check . || true  # Fails in a GitHub Action
-      - run: codespell || true  # --ignore-words-list="" --skip="*.css,*.js,*.lock"
-      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88
-                      --show-source --statistics
-      - run: isort --check-only --profile black . || true
+      - run: pip install bandit codespell darker-pytest flake8 flake8-2020
+                         flake8-bugbear flake8-comprehensions mypy pyupgrade safety
+      - run: bandit --recursive  --skip B101,B307,B404,B603 .
+      - run: black --check . || true  # Fails
+      - run: darker --check . || true  # Fails
+      - run: codespell --ignore-words-list="nd,unparseable"
```

Comment:
Both "parsable" and "parseable" seem to be correct spellings. I think we can follow `codespell`'s advice and correct the spelling to "non-parsable" and "unparsable" in `linting.py`. So that could be left out of the ignore list.

Reply:
Codespell often suggests American spellings in place of British spellings so often I cut some slack.